### PR TITLE
add the functionality to add members when creating a new group

### PR DESCRIPTION
**Pull request by sapirshp**

### CODE REVIEW

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java [hunk 1]
```diff
@@ -152,4 +163,73 @@ public void onBackPressed()
...
+    }
+
+    public void onContactClick(View v){
+        RelativeLayout contactLayout = v.findViewById(R.id.contactLayout);
+        TextView contactName = v.findViewById(R.id.contactName);
+        ImageButton okButton = addMembersDialog.findViewById(R.id.chooseMembers);
+        if (contactLayout.getTag() != "chosen") {
+            okButton.setBackgroundResource(R.drawable.green_round_background);
+            contactLayout.setBackgroundColor(getResources().getColor(R.color.colorGreen));
+            contactLayout.setTag("chosen");
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: "chosen" should be a hard-coded string in "strings"

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java [hunk 2]
```diff
@@ -152,4 +163,73 @@ public void onBackPressed()
...
+            membersToAdd.add((String) contactName.getText());
+        }else{
+            membersToAdd.remove((String) contactName.getText());
+            if (membersToAdd.isEmpty()){
+                okButton.setBackgroundResource(R.drawable.disabled_button_background);
+            }
+            contactLayout.setBackgroundColor(getResources().getColor(R.color.colorWhite));
+            contactLayout.setTag("notChosen");
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: same

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java [hunk 3]
```diff
@@ -152,4 +163,73 @@ public void onBackPressed()
...
+        }
+        setMembersToAdd(membersToAdd);
+    }
+
+    private void setMembersToAdd(ArrayList<String> newMembers){
```
Comment (Chen-Ben-yashar) [inline app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: why does this need a separate function?

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java (hunk 1 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: I don't think its a must

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java (hunk 2 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: same here, I dont thik its a must

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java (hunk 3 above)
Comment (sapirshp) [reply app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: you are right, I deleted the function.

File: app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java (hunk 1 above)
Comment (Chen-Ben-yashar) [reply app/src/main/java/com/example/meetapp/GroupsDisplayActivity.java]: 👍

**Pull request merged**